rocAL/rocAL/source/device/ocl_setup.cpp

```diff
@@ -67,34 +64,30 @@ int get_device_and_context(int devIdx, cl_context *pContext, cl_device_id *pDevi
             const std::string AMD_NAME = "Advanced Micro Devices, Inc.";
 
             if (status != CL_SUCCESS) {
-                LOG("clGetPlatformInfo returned error: "+TOSTR(status))
+                LOG("clGetPlatformInfo returned error: " + TOSTR(status))
                 continue;
             }
-            if( AMD_NAME.compare(0, AMD_NAME.length(), vendor) == 0)
-            {
-                LOG("AMD Platform found "+STR(vendor))
+            if (AMD_NAME.compare(0, AMD_NAME.length(), vendor) == 0) {
+                LOG("AMD Platform found " + STR(vendor))
                 platform = platforms[i];
                 break;
             }
-
         }
     }
 
-    if(!platform)
+    if (!platform)
         THROW("Couldn't find AMD OpenCL platform")
 
-
     // enumerate devices
 
     // To Do: handle multi-GPU case, pick appropriate GPU/APU
     char driverVersion[100] = "\0";
 
     cl_context_properties contextProps[3] =
-    {
+        {
```

Comment:
@fiona-gladwin move { one line up

Reply:
The C++ formatter resulted in this kind of formatting. Since it is not a function definition